ner_v1/api.py

```diff
@@ -9,12 +9,12 @@
     get_number, get_passenger_count, get_shopping_size, get_time, get_time_with_range, get_date, get_budget, \
     get_person_name, get_regex
 from ner_v1.chatbot.tag_message import run_ner
-from ner_v1.constant import PARAMETER_MESSAGE, PARAMETER_ENTITY_NAME, PARAMETER_STRUCTURED_VALUE, \
```

Comment:
1) No need for separate constant directory. At the main level, (outside ner v1 and ner v2), make a file called constant.py which will contain content of ner_constant.py.
2) Let other detector_constant file be inside ner_v1 and make a separate one for ner_v2. Let that also be constant.py
3) Shift number detection, text detection and name detection constant inside ner_v1 file.

Reply:
![screen shot 2018-10-24 at 10 59 07 am](https://user-images.githubusercontent.com/10032107/47408470-2f584d00-d77d-11e8-9529-68fff60572fa.png)

Reply:
Done